packages/eas-cli/src/commands/channel/create.ts

```diff
@@ -17,11 +17,11 @@ import { createUpdateBranchOnAppAsync } from '../branch/create';
 
 async function createUpdateChannelOnAppAsync({
   appId,
-  channelName,
+  name,
   branchId,
```

Comment:
i think this is fine, but it's less obvious that `name` is `channelName` in `createUpdateChannelOnAppAsync`, compared to the other rename, where it's pretty clear that `name` is `branchName` in `createBranchAsync`

Reply:
yea that's a good point, Lemme see if it makes sense to switch it back.

Reply:
switched!